### ISSUE TITLE
Forms search: Update 'read more' links in the blue boxes after wizards are moved

### ISSUE THREAD

**Issue by jenniferlee-dsva** [department-of-veterans-affairs/va.gov-team#12535]
## Update 'read more' links in the blue boxes after wizards are moved

After the wizards for the disability and education applications have been moved to their respective online application pages, we need to point the form links from the content page to the application page. 

### Tasks
- [ ] **Disability box:**
    - 1/ Change `Read more` text link to: Apply online
    - 2/ Change the link to point to the 526ez online tool intro page.
- [ ] **Education box:**
    - 1/ Change `Read more` text link to: Apply online
    - 2/ Change the link to point to the 1990 online tool intro page.
- [x] **Health care box:**
    - 1/ Change `Read more` text link to: Apply online
    - _Note:_ This link is already pointing to the online app because the health care app doesn't have a wizard. Therefore no change here other than text update.

### Acceptance criteria
- [ ] Blue box on /find-forms includes a link to 526ez online tool intro page with updated text "apply online"
- [ ] Blue box on /find-forms includes a link to 1990 online tool intro page with updated text "apply online"
- [x] Blue box on /find-forms that links to apply for health care has updated text "apply online"

**Screenshot:** 
![image](https://user-images.githubusercontent.com/43381063/90415891-bc217880-e07f-11ea-8954-72b5a39bca90.png)

**Comment by kelsonic** [comment]
Do we know what the links will be here for Disability + Education boxes? 🙂

**Comment by jenniferlee-dsva** [comment]
@kelsonic - It depends if the URLs will have changed, once the wizards have been moved. I believe they will stay the same, so the URLs would be: 

https://www.va.gov/disability/file-disability-claim-form-21-526ez/introduction

https://www.va.gov/education/apply-for-education-benefits/application/1990

Note: We can't point to the application start pages until after the wizards are moved.  The above URLs should be confirmed however **after the wizards have been moved.**

**Comment by jenniferlee-dsva** [comment]
FYSA - If the disability and education wizards work is blocked, we should separate out the health care box update and work on that separately.

**Comment by ncksllvn** [comment]
@jenniferlee-dsva I updated the issue description to use a task list so we can do the health care update without a separate issue. I haven't been following the wizard migrations very well so I will likely be dependent on other folks to let us know when we are unblocked for the other two

**Comment by ncksllvn** [comment]
I have a PR open to make this change @jenniferlee-dsva -

![image](https://user-images.githubusercontent.com/1915775/94462368-6b766280-0189-11eb-94d7-d6e973a620d8.png)

Look okay for merge?

**Comment by ncksllvn** [comment]
PR for updating the `Apply for VA health care` blue box will go out today

**Comment by jenniferlee-dsva** [comment]
Lgtm, thanks @ncksllvn

**Comment by MarciMcGuireGCIO** [comment]
Assigned to Decision Tools team and added dt-find-va-form label.

**Comment by MarciMcGuireGCIO** [comment]
Pulling this out of the https://github.com/department-of-veterans-affairs/va.gov-team/issues/10758 epic since there's a dependency on PW, and go-live date for that work is yet unknown.

**Comment by zacharymorel** [comment]
Per @johnhashva Education launch late January, Va Benefits launch late February,

**Comment by MarciMcGuireGCIO** [comment]
> Per @johnhashva Education launch possibly late January, Va Benefits launch possibly late February,
Leaving this issue blocked due to the remaining dependencies on _How to Apply_ Wizards.  Will re-visit in a month to check if things are still on track.

**Comment by zacharymorel** [comment]
That sounds good to me @MarciMcGuireGCIO .

**Comment by zacharymorel** [comment]
Given an estimate now that the discovery is done.

**Comment by MarciMcGuireGCIO** [comment]
@johnhashva I had a note to follow up with you on the status of the _How to Apply_ wizards so we could determine if this is still blocked.

**Comment by johnhashva** [comment]
@MarciMcGuireGCIO getting a fresh update from the product lead -- but here is where things stand based on some background movement:  the work that will result in a change to the link for File a disability claim is in staging review.   Might be live in the next few weeks.   Work for the Apply for the GI Bill is still in (slow) progress.  

-

**Comment by MarciMcGuireGCIO** [comment]
Thanks, @johnhashva !

**Comment by MarciMcGuireGCIO** [comment]
Pulling this out of Blocked since the How to Apply wizards change have now been deployed.  We will prioritize in our next Grooming meeting on 7/21.

**Comment by zacharymorel** [comment]
@MarciMcGuireGCIO Do we have links of the new wizards?

**Comment by MarciMcGuireGCIO** [comment]
@brianalloyd -- this ticket involves updating some links to point to new "how to apply" wizards.  Can you possibly confirm that the links below are correct?  Thanks! cc @zacharymorel 

- 21-526ez: https://www.va.gov/disability/file-disability-claim-form-21-526ez/introduction
- 22-1990: https://www.va.gov/education/apply-for-education-benefits/application/1990/introduction
- 10-10ez: (no change) https://www.va.gov/health-care/apply/application/introduction

**Comment by MarciMcGuireGCIO** [comment]
@brianalloyd @kelsonic I know you both have a lot going on. Zach needs to make this quick change to finish this task, and we'd just like someone to verify we have the right links in my comment above. TIA!   cc @zacharymorel

**Comment by kelsonic** [comment]
Those look correct to me! ✅ but I would definitely confirm with @brianalloyd or maybe someone on the content team such as @RLHecht or @DanielleThierryUSDSVA 🙂

**Comment by MarciMcGuireGCIO** [comment]
Thanks, @kelsonic !

**Comment by RLHecht** [comment]
@MarciMcGuireGCIO I believe these should all have the /introduction part removed from the url. @mnorthuis explained to me that the FE will resolve the appropriate code so the links should also just go to the primary form url. Mikki, can you confirm?

**Comment by MarciMcGuireGCIO** [comment]
Changing to Sprint 56 since we need confirmation before making this change.

**Comment by mnorthuis** [comment]
That is correct, we should eliminate the /introduction where possible since that isn't always the first page of the form flow.  Some forms now have a /start prior to the introduction page.  Great catch @RLHecht!

Also, the wizard has not moved for education forms, so should still be directing users to the "How to apply" page for that. 

Question:  I don't believe "Read more" is inclusive, nor is it technically a correct label.  I believe it should say something like "Learn how to apply" for both inclusivity and accuracy to where it is taking users.

**Comment by zacharymorel** [comment]
@MarciMcGuireGCIO This is ready to go on Staging.

**Comment by MarciMcGuireGCIO** [comment]
Looks great @zacharymorel!

**Comment by zacharymorel** [comment]
Once this is on Production, this is fine to close, yes?

**Comment by MarciMcGuireGCIO** [comment]
Yep!😁

**Comment by zacharymorel** [comment]
@MarciMcGuireGCIO This is live on prod.

**Comment by MarciMcGuireGCIO** [comment]
@zacharymorel - awesome! Looks great!  Cc @davidconlon

**Comment by MarciMcGuireGCIO** [comment]
@zacharymorel - this change is actually not on Prod, but that's OK because we need to make a small tweak. 

The wizard for the GI bill is actually not live yet, so we need to change that URL back to  https://www.va.gov/education/how-to-apply/, and the label should say "Learn how to apply online >"   

Thanks!!

**Comment by zacharymorel** [comment]
Oh thank you for catching that @MarciMcGuireGCIO .

**Comment by MarciMcGuireGCIO** [comment]
Actually it was Mikki who caught it!  😺

**Comment by zacharymorel** [comment]
I'm on it!

**Comment by MarciMcGuireGCIO** [comment]
Thank you, sir! 🎩

**Comment by zacharymorel** [comment]
@MarciMcGuireGCIO This should be fixed on staging now

**Comment by MarciMcGuireGCIO** [comment]
@zacharymorel this looks spot-on to me.  FYSA @mnorthuis 

![image](https://user-images.githubusercontent.com/73354907/133150907-0c73a104-9b4e-4487-8004-cf261ba38857.png)

**Comment by MarciMcGuireGCIO** [comment]
@zacharymorel this didn't get to Production, just FYI. Thx!

**Comment by zacharymorel** [comment]
> @zacharymorel this didn't get to Production, just FYI. Thx!

It will be today!

**Comment by MarciMcGuireGCIO** [comment]
🥳

**Comment by zacharymorel** [comment]
@MarciMcGuireGCIO This is on Prod now!

**Comment by MarciMcGuireGCIO** [comment]
Whoo hoo!!  Thanks, @zacharymorel!   Looks  great!  And I am actually looking at Prod this time and not Staging!
